pkg/auth/auth.go

```diff
@@ -32,7 +30,9 @@ const (
 	GROUNDCOVER_AUTH_PATH                  = ".groundcover"
 	GROUNDCOVER_AUTH_FILE                  = "auth.json"
 	GROUNDCOVER_API_KEY_FILE               = "api_key.json"
-	AUTH0_MIN_INTERVAL                     = time.Second * 5
+	SPINNER_TYPE                           = 26 // ....
```

Comment:
why changing the spinner type?

Reply:
I think the universal animation for waiting is (. .. ... )